articles/api/management/v2/tokens-flows.md

```diff
@@ -49,11 +49,10 @@ Having a token that never expires can be very risky, in case an attacker gets ho
 
 ## Can I still get a non-expiring token?
 
-Yes you can. We added a text box (__Token Expiration (Seconds)__), at [the API Explorer tab of your Auth0 Management API](${manage_url}/#/apis/management/explorer), where you can set the new expiration time (in seconds) and click __Update & Regenerate Token__. A new token will be generated with your custom expiration time. Our recommendation however is not to use this and get a new token every 24 hours. You can easily automate this [following this process](/api/management/v2/tokens).
+Yes, you can. We added a text box (__Token Expiration (Seconds)__) on [the API Explorer tab of your Auth0 Management API](${manage_url}/#/apis/management/explorer), which allows you to set the new expiration time (in seconds), and click __Update & Regenerate Token__. A new token will be generated with your custom expiration time. Our recommendation, however, is not to use this and get a new token every 24 hours. You can easily automate this [following this process](/api/management/v2/tokens).
```

Comment:
I would not even mention this since is a security issue bound to happen if token gets leaked. We should recommend following the process mentioned in `/api/management/v2/tokens`